prow/config.yaml

```diff
@@ -4758,7 +4757,7 @@ presubmits:
     max_concurrency: 0
     name: pull-security-kubernetes-local-e2e-containerized
     rerun_command: /test pull-security-kubernetes-local-e2e-containerized
-    run_if_changed: ^(pkg|cmd)/kubelet.*$
+    run_if_changed: ""
```

Comment:
drop

Reply:
i tried the ... verify script yelled at me

Reply:
@krzyzacy